sebs/fission/minio.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python3
+from sebs.faas.storage import PersistentStorage
+from typing import List, Tuple, Any
+import logging
+from time import sleep
+import minio
+import secrets
+import os
+
+
+class Minio(PersistentStorage):
```

Comment:
We should replace it with a generic implementation of Minio - look at OpenWhisk. There should be no need to specialize.

Reply:
done